Classes/ToolRegistrator.lua

```diff
@@ -297,7 +303,17 @@ local function LeftClickToolsHook(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_Bloc
 	end
 
 	local State = GetPlayerState(a_Player)
-	return State.ToolRegistrator:UseLeftClickTool(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, a_Player:GetEquippedItem().m_ItemType)
+	State.ToolRegistrator:UseLeftClickTool(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, a_Player:GetEquippedItem().m_ItemType)
+end
+
+
+
+
+
+local function PlayerBrokeBlockHook(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, a_BlockType, a_BlockMeta)
```

Comment:
Unless there's some reason why it wouldn't work, this should be PlayerBreakingBlockHook and use HOOK_PLAYER_BREAKING_BLOCK, then just return true when the itemtype is a wooden axe to cancel the action.

Reply:
> Unless there's some reason why it wouldn't work, this should be PlayerBreakingBlockHook and use HOOK_PLAYER_BREAKING_BLOCK, then just return true when the itemtype is a wooden axe to cancel the action.

This causes a very noticeable frieze which I tried to fix initially.